hawk/src/test/java/com/orhanobut/hawk/DataHelperTest.java

```diff
@@ -34,7 +34,7 @@
   }
 
   @Test public void testOldVersionCheck() {
-    DataInfo info = DataHelper.getDataInfo("java.lang.String##00@asdfjasdf");
```

Comment:
reading the old implementation of the `addType` method, the String had a different format ( https://github.com/orhanobut/hawk/blob/eaf420196396c341b62f7ced6c86df207cf8f91c/hawk/src/main/java/com/orhanobut/hawk/DataUtil.java#L65 ). Am I missing something?